apps/android/common/src/commonMain/kotlin/chat/simplex/common/views/usersettings/UserProfileView.kt

```diff
@@ -199,29 +198,28 @@ fun EditImageButton(click: () -> Unit) {
 fun DeleteImageButton(click: () -> Unit) {
   IconButton(onClick = click) {
     Icon(
-      painterResource(R.drawable.ic_close),
-      contentDescription = stringResource(R.string.delete_image),
+      painterResource(MR.images.ic_close),
```

Comment:
was it necessary to rename R to MR?

Reply:
renamed back just for review. And yes, it's necessary for the lib we use since it's not an Android plugin anymore